src/backend/joanie/settings.py

```diff
@@ -72,6 +72,8 @@ class Base(Configuration):
     # Security
     ALLOWED_HOSTS = values.ListValue([])
     SECRET_KEY = values.Value(None)
+    # Security - Server to server authorized API keys
+    AUTHORIZED_API_TOKENS = values.ListValue([])
```

Comment:
By convention, we prefix all settings related to joanie with `JOANIE_`.
So I'll rename this setting `JOANIE_AUTHORIZED_API_TOKENS`.

Then you should set environ_prefix to None otherwise I believe the environment variable must be prefixed by DJANGO.

```suggestion
    JOANIE_AUTHORIZED_API_TOKENS = values.ListValue([], environ_prefix=None)
```